sopel/plugins/handlers.py

```diff
@@ -221,7 +223,13 @@ class PyModulePlugin(AbstractPluginHandler):
         True
 
     """
+
     PLUGIN_TYPE = 'python-module'
+    """Plugin's type.
+
+    Metadata for the plugin, this should be considered to be a constant and
+    should not be modified at runtime.
+    """
```

Comment:
```suggestion
    """The plugin's type.

    Metadata for the plugin; this should be considered to be a constant and
    should not be modified at runtime.
    """
```

Aside from grammar, is this important enough to warrant an `.. important::` block?

Reply:
Between the naming convention and the fact that it's all internal stuff, I don't worry that much.